admin-center/display/govbox-health.md

```diff
@@ -1,12 +1,21 @@
 ---
-layout: post
+layout: page
 title: Help Searchers Find Health Information
 category: manual
 tags: help-manual display modules
+date: February 27, 2017
+last_modified_at: February 27, 2017
+sidenav: admin_center
+redirect_from:
+    - /manual/govbox-health.html
 ---
 
+Find it in the Admin Center: [Search.gov Home]({{ site.baseurl }}/index.html) > [Admin Center](https://search.usa.gov/sites/) > YourSite > Manage Display > Display Overview
+
 Searches for health-related information are very common. We created the Health Topics module to help searchers find high-quality, authoritative health information.
 
 This module offers direct access to [MedlinePlus Health Topics](https://www.nlm.nih.gov/medlineplus/healthtopics.html) and open clinical studies and trials on [ClinicalTrials.gov](https://clinicaltrials.gov/). To provide timely information across all health topics, we pull the data from the [MedlinePlus XML files](https://www.nlm.nih.gov/medlineplus/xml.html), which cover both the English and Spanish health topics.
 
 If you [opt to show the Health Topics]({{ site.baseurl }}/manual/display-overview.html) module, searchers looking for health topics on your site can readily find high-quality health information.
+
+If you would like to show [health topics]({{ site.baseurl }}/manual/govbox-health.html) in your results, but do not see the toggle to turn them on, [email us](mailto:search@support.digitalgov.gov) for assistance.
```

Comment:
same as for jobs and federal register, I realized this link points back to this very same page. Please remove.